synapse/models/orgs.py

```diff
@@ -1123,6 +1139,39 @@ def getModelDefs(self):
                     }),
                     # TODO duration ('duration'
                 )),
+                ('ou:requirement', {}, (
+
+                    ('name', ('str', {'lower': True, 'onespace': True}), {
+                        'doc': 'A name for the requirement.'}),
+
+                    ('text', ('str', {}), {
+                        'disp': {'hint': 'text'},
+                        'doc': 'The text of the stated requirement.'}),
+
+                    ('priority', ('meta:priority', {}), {
+                        'doc': 'The priority of the requirement.'}),
+
+                    ('level', ('int', {'enums': reqlevels}), {
```

Comment:
"Level" of requirements is unclear to me, especially with the defined taxonomy. Level of requirements commonly refers to its level of abstraction (business, product, feature, etc.). Criticality? Importance?

Reply:
This is the property where the `should` / `shall` gets mapped in as an enumeration to allow for queries like `level>=should`. If it was an arbitrary taxonomy, that type of linear `>=` query would not be possible, and in general i think we can probably boil down requirements to `desired` / `should` / `shall` primitives. Open to other names, but it's different than `priority` in my mind :)